packages/db/src/models/execution.ts

```diff
@@ -13,6 +13,15 @@ class ExecutionPgModel extends BasePgModel<PostgresExecution, PostgresExecutionR
     });
   }
 
+  async create(
+    knexOrTransaction: Knex | Knex.Transaction,
+    execution: PostgresExecution
+  ) {
+    return await knexOrTransaction(this.tableName)
+      .insert(execution)
+      .returning('*');
```

Comment:
Same sugestion from the other PR:

There's nothing wrong with this, but since we're going to need to do the same thing for every data type, I think we should update the base model create to have an optional argument for returningFields, which could be a string or array of strings. Then from this method (and for all the other data type overrides we're going to do) we could do:

`return super.get(knexOrTransaction, item, '*')`

Reply:
[`bd0112b` (#2422)](https://github.com/nasa/cumulus/pull/2422/commits/bd0112b50cc0fda564ec0be0a84a830659daf1ea)